objects/scene-objects/group.md

```diff
@@ -1,21 +1,7 @@
 # Group
 
-The primary function of the **Group** **Node** is that it allows us to transform multiple **Objects**, relative to a single **Pivot Point**, as if though they are a single, unified **Object**.
+The primary function of the **Group** **Object** is that it allows us to transform multiple **Objects**, relative to a single **Pivot Point**, as if though they are a single, unified **Object**.
 
 Additionally, it is a useful way of organizing collections of **Objects** into categories, and making **Scenes** with high amounts of different **Objects**, more manageable.
```

Comment:
Remove this as I merged it into the above suggestion

Reply:
you mean
*Additionally, it is a useful way .... ... *